Added converter for OffsetDateTime object

beuvenar opened the pull request.

Commit:
Added converter for OffsetDateTime object

ufoscout commented on jpo-commons-types/src/main/java/com/jporm/types/ext/OffsetDateTimeToLocalDateTimeTimestampConverter.java:
```diff
@@ -0,0 +1,48 @@
+	
+	@Override
+    public OffsetDateTime clone(final OffsetDateTime source) {
+		return OffsetDateTime.of(source.toLocalDateTime(), source.getOffset());
```
Do not clone immutable objects, return the object itself

Commit:
- Return same instance in clone method of OffsetDateTimeToLocalDateTimeTimestampConverter.java since OffsetDateTime is immutable
- Adapted Javadoc of clone method in TypeConverter to indicate that the same instance can be returned if the object is immutable
